src/screens/Login.js

```diff
@@ -8,25 +8,30 @@ function Login({ navigation }) {
   const [callbacks, setCallbacks] = useState([]);
   const [nxt, setNxt] = useState(null);
 
-  useEffect(() => {
-    async function logout() {
-      await ForgeRockModule.performUserLogout();
-    }
-    logout();
-  }, []);
-
   useEffect(() => {
     (async () => {
-      await ForgeRockModule.frAuthStart();
-      const data = await ForgeRockModule.loginWithoutUI();
-      const next = JSON.parse(data);
-      setNxt(next);
-      setCallbacks(
-        next.callbacks.map((res) => ({
-          ...res,
-          response: JSON.parse(res.response),
-        })),
-      );
+      try {
+        const data = await ForgeRockModule.loginWithoutUI();
```

Comment:
@george-bafaloukas-forgerock Is the method `loginWithoutUI` a method name that comes for the Core SDKs, or is that a name we added in the bridge code?